tests/proxy/proxy.md

```diff
@@ -1,5 +1,4 @@
-Using a proxy for mocked data
----
+## Using a proxy for mocked data
```

Comment:
I wondered why it was formatted to `##` not to `#`.
It seems `#` is valid for prettier, but the next level used in this document is `###`.
So, I guess `##` is fine as a top level.

Reply:
That's because previously we were using `---` which means "heading level 2". See https://www.markdownguide.org/basic-syntax/#alternate-syntax.

I'll fix this manually.

Reply:
I've fixed this in https://github.com/woocommerce/google-listings-and-ads/pull/876/commits/e17304b276bacb3d05c713adee017df5780e58a0.

Reply:
Thanks! :) I thought `----` and `=====` are equivalent :facepalm: